src/lib/database/seed.ts

```diff
@@ -95,6 +95,7 @@ export const seed = (db: Database): Seed => {
     properties: {
       is_cooling_available: true,
       is_heating_available: true,
+      min_heating_cooling_delta_celsius: 8,
```

Comment:
```suggestion
      min_heating_cooling_delta_celsius: 2.78,
```

I think typically for ecobee this is 5 degrees fahrenheit